cfme/tests/control/test_actions.py

```diff
@@ -220,6 +221,17 @@ def _finalize():
 
 
 @pytest.fixture(scope="module")
+def vm(request, provider, local_setup_provider, small_template_modscope, vm_name):
+    return _get_vm(request, provider, small_template_modscope, vm_name)
+
+
+@pytest.fixture(scope="module")
+def vm_big(request, provider, local_setup_provider, big_template_modscope, vm_name):
+    vm_name = "{}_big".format(vm_name)
```

Comment:
Dima, you will want to use this instead: https://github.com/ManageIQ/integration_tests/blob/master/utils/generators.py#L6

We want to avoid underscores in VM names because they cause issues with certain providers.